lib/serviceDiscovery/helpers.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const os = require('os');
+const crypto = require('crypto');
+const memoizee = require('memoizee');
+
+const interfaces = os.networkInterfaces();
+const ifaceNames = Object.keys(interfaces);
+
+// choose the first valid ip as the one we will announce
```

Comment:
I can't tell what this comment applies to... Not to the `let announceIps` below I assume.

Reply:
No idea. Guessing that was a comment from a prior prototype implementation I had... apologies for the noise.